charts/cryostat/tests/service_test.yaml

```diff
@@ -0,0 +1,62 @@
+suite: test service.yaml
+templates:
+  - service.yaml
+
+tests:
+  - it: should create a Service with the correct settings
+    set:
+      core.service.type: ClusterIP
+      core.service.httpPort: 8080
+      authentication.openshift.enabled: false
+    asserts:
+      - hasDocuments:
+          count: 1
+      - equal:
+          path: kind
+          value: Service
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-cryostat
+      - equal:
+          path: spec.type
+          value: ClusterIP
+      - equal:
+          path: spec.ports[0].port
+          value: 8080
+      - equal:
+          path: spec.ports[0].targetPort
+          value: 4180
+      - equal:
+          path: spec.ports[0].protocol
+          value: TCP
+      - equal:
+          path: spec.ports[0].name
+          value: cryostat-http
```

Comment:
We also check if `cryostat-https` port is available when openshift auth is enabled.

Reply:
See #167